sdk/core/azure-core/azure/core/pipeline/policies/_base_async.py

```diff
@@ -48,14 +48,13 @@ class AsyncHTTPPolicy(abc.ABC, Generic[HTTPRequestTypeVar, AsyncHTTPResponseType
     :type next: ~azure.core.pipeline.policies.AsyncHTTPPolicy or ~azure.core.pipeline.transport.AsyncHttpTransport
     """
 
-    next: Union[
-        "AsyncHTTPPolicy[HTTPRequestTypeVar, AsyncHTTPResponseTypeVar]",
-        "AsyncHttpTransport[HTTPRequestTypeVar, AsyncHTTPResponseTypeVar]",
```

Comment:
This was an error introduced in my previous PR. `next` is always a policy, and can never be a transport (the transport is wrapped in a TransportRunner that implements a policy protocol)